<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLarik</class>
 <widget class="QWidget" name="FormLarik">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>142</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID Pengguna</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nama Larik</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EditNama"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Info Lokasi</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="EditLokasi"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EditId"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="EditIdLarik"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ID Larik</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_Hapus">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Simpan">
      <property name="text">
       <string>Simpan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Edit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
